e2e/test/scenarios/sharing/public-resource-downloads.cy.spec.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  ORDERS_DASHBOARD_DASHCARD_ID,
+  ORDERS_DASHBOARD_ID,
+  ORDERS_BY_YEAR_QUESTION_ID,
+} from "e2e/support/cypress_sample_instance_data";
+import {
+  assertNotEmpty,
```

Comment:
```suggestion
  assertNotEmptyObject,
```

We're also not using this function in `embed-resource-downloads.cy.spec.ts` should we have the same assertions?